src/main/java/life/qbic/portal/portlet/OMEROClientPortlet.java

```diff
@@ -128,42 +131,68 @@ private Panel getImgViewer() {
         VerticalLayout projectLayout = new VerticalLayout();
         projectLayout.setSpacing(true);
         projectLayout.setMargin(false);
-        projectLayout.setWidth("50%");
+        projectLayout.setWidth("100%");
         projectLayout.setHeight("100%");
 
         //HorizontalLayout topPanelLayout = new HorizontalLayout();
-        GridLayout topPanelLayout = new GridLayout(3, 1);
+        //GridLayout topPanelLayout = new GridLayout(3, 1);
+        GridLayout topPanelLayout = new GridLayout(2, 4);
         topPanelLayout.setSpacing(true);
         topPanelLayout.setMargin(false);
         topPanelLayout.setWidth("100%");
         topPanelLayout.setHeight("100%");
 
-        projectBox = new ComboBox<>("Select project:");
-        projectBox.setEmptySelectionAllowed(false);
-        projectBox.setDataProvider(new ListDataProvider<>(projects));
-        projectBox.setItemCaptionGenerator(Project::getName);
-        projectBox.setWidth("100%");
+        ////////////
+        // project grid
+        projectGrid = new Grid<>();
+
+        ListDataProvider<Project> projectListDataProvider = new ListDataProvider<>(projects);
+        projectGrid.setDataProvider(projectListDataProvider);
+        projectGrid.setSelectionMode(SelectionMode.SINGLE);
+        projectGrid.setCaption("Projects");
+        projectGrid.setSizeFull();
+        projectGrid.setWidth("100%");
+        projectGrid.setHeight("300px");
+
+        Column<Project, String> projectCodeColumn = projectGrid.addColumn(Project::getName).setCaption("Project Code");
+        Column<Project, String> projectDescColumn = projectGrid.addColumn(Project::getDescription).setCaption("Project Description");
+        projectCodeColumn.setWidth(150);
+        projectDescColumn.setWidth(700);
```

Comment:
Good point, it would be neat to talk about it down the line since I think the other grids might benefit from having such a setup. So far, for the column object, I have only found the setWidth() method that takes a float for the number of pixels.
Perhaps I will open an issue here since here I might have to look at all the grid components.